repositories/testimonial-repository.js

```diff
@@ -0,0 +1,77 @@
+const db = require("../models/index");
+const pick = require("../functions/pick");
+
+const createTestimonial = async (req) => {
+  let { name, image, content } = req.body;
+
+  const create = await db.sequelize.transaction(async (t) => {
+    const [testimonial, wasCreated] = await db.Testimonial.findOrCreate({
+      where: { name },
```

Comment:
I think this condition won't let two different people with the same name had their own testimonial

Reply:
Fixed it by replacing `findOrCreate` with the `create` method. Thanks